CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # DrawerKit
 
+## v. 0.7.0
+
+- DrawerKit now supports interactions with presenting view. By default presenting view will receive touch events when drawer is in `collapsed` or `partiallyExpanded` state. This can be configured via new `passthroughTouchesInStates` configuration.
```

Comment:
```suggestion
- DrawerKit now supports interactions with the presenting view. By default, the presenting view will receive touch events when the drawer is in the `collapsed` or `partiallyExpanded` state. This can be configured via the new `passthroughTouchesInStates` configuration.
```